src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -1159,127 +1162,109 @@ public void updateInventory()
 	@Override
 	public void incrementStatistic(@NotNull Statistic statistic)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		incrementStatistic(statistic, 1);
 	}
 
 	@Override
 	public void decrementStatistic(@NotNull Statistic statistic)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		decrementStatistic(statistic, 1);
 	}
 
 	@Override
 	public void incrementStatistic(@NotNull Statistic statistic, int amount)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		statistics.setStatistic(statistic, statistics.getStatistic(statistic) + amount);
 	}
 
 	@Override
 	public void decrementStatistic(@NotNull Statistic statistic, int amount)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		statistics.setStatistic(statistic, statistics.getStatistic(statistic) - amount);
```

Comment:
two operations for what can be done in one, I suggest you to add a `increment` and `decrement` method in your `StatisticsMock` and implement the logic there.